packages/synapse-sdk/src/types.ts

```diff
@@ -404,14 +404,23 @@ export interface PreflightInfo {
  * 2. Piece addition (transaction submitted to chain)
  * 3. Confirmation (transaction confirmed on-chain)
  */
+export interface PieceIdentifiers {
```

Comment:
should this be PiecePair removed entirely? `{ pieceCid: PieceCID; pieceId: number }` is used in quite a few places, a good name for this representation of a Piece could be useful..

if the core maintainers disagree we could just remove this.